deploy/kubevirt-hostpath-provisioner.yaml

```diff
@@ -28,10 +28,12 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: kubevirt-hostpath-provisioner
 rules:
+  - apiGroups: [""]
```

Comment:
Any reason we need this permission? We already know which node we are on with the daemonSet.

Reply:
Without this it will fail on the dynamic flow here:
https://github.com/kubevirt/hostpath-provisioner/blob/master/controller/controller.go#L1243

Reply:
Interesting, I guess we never hit that part of the code before.